style/layer/boundary.js

```diff
@@ -178,6 +178,7 @@ export const countryCasing = {
   filter: ["all", ["in", "admin_level", 2], ["==", "maritime", 0]],
   minzoom: 2,
   layout: {
+    "line-cap": "round",
```

Comment:
Especially along disputed boundaries, the boundary lines can be split up into multiple LineString features that OpenMapTiles doesn’t union together, due to differing tags like `claimed_by` and `disputed_by`. These splits result in gaps that need to be filled in with line caps. A round line cap won’t look great with a translucent line, but overlaps aren’t noticeable at the zoom levels where the line would be translucent.

<img src="https://user-images.githubusercontent.com/1231218/156655238-4d89af1d-862a-4457-8f03-8e0fce92408f.png" width="400" alt="line-cap">